### PR TITLE
Add DetectionVOCEvaluator to train script of Faster R-CNN

### PR DESCRIPTION
**Pull request by yuyu2172**

### CODE REVIEW

File: examples/faster_rcnn/train.py [hunk 1]
```diff
@@ -104,6 +110,12 @@ def transform(in_data):
             trigger=plot_interval
         )
 
+    trainer.extend(
+        DetectionVOCEvaluator(
+            test_iter, model.faster_rcnn, use_07_metric=True),
+        trigger=(args.step_size, 'iteration'),
+        invoke_before_training=False)
+
```
Comment (Hakuyume) [inline examples/faster_rcnn/train.py]: I guess this evaluator runs only once. Is it OK?

Comment (yuyu2172) [inline examples/faster_rcnn/train.py]: Yes

Comment (Hakuyume) [inline examples/faster_rcnn/train.py]: How about running evaluator twice (when lr changes and when the training finishes)?

Comment (yuyu2172) [inline examples/faster_rcnn/train.py]: OK.

**Comment by Hakuyume** [comment]
LGTM

**Pull request merged**